specifyweb/frontend/js_src/lib/components/DataModel/interactionBusinessRules.ts

```diff
@@ -55,15 +53,11 @@ export const getTotalResolved = (
   loanReturnPrep: SpecifyResource<LoanReturnPreparation>
 ) => {
   return loanReturnPrep.collection
-    ? _.reduce(
-        loanReturnPrep.collection.models,
-        (sum, loanPrep) => {
-          return loanPrep.cid != loanReturnPrep.cid
-            ? sum + loanPrep.get('quantityResolved')
-            : sum;
-        },
-        0
-      )
+    ? loanReturnPrep.collection.models.reduce((sum, loanPrep) => {
+        return loanPrep.cid != loanReturnPrep.cid
```

Comment:
no need for explicit `{`, `return` and `}`, can just remove them to do implicit return

Reply:
it was needed earlier because shorter syntax did not exist. Please consider also going though `businessRules.ts` to remove them